biome.jsonc

```diff
@@ -52,6 +52,10 @@
     "enabled": true,
     "rules": {
       "recommended": true,
+      "suspicious": {
+        // https://developer.mozilla.org/en-US/docs/Web/API/CookieStore Reevaluate in June '26 after 1y of availability.
```

Comment:
Maybe add a TODO + Jira to improve our chances of actually re-evaluating this 😁

Reply:
- [ ] Jira & todo